### DISCUSSION TITLE
New Enterprise Tier + SSO

### DISCUSSION THREAD

**Discussion by ghost** [BerriAI/litellm#2023] [Announcements]
Hi everyone, 

We've launched a [new enterprise folder](https://github.com/BerriAI/litellm/tree/main/enterprise) on the repo. 

### Am I impacted? 

Do you use SSO? 

- Yes => Then, not for 6 months - next Friday (02/23), SSO will move into the enterprise tier and will remain free for all existing users for the next 6 months (08/23).  [Let us know, if you're impacted](https://calendly.com/d/4mp-gd3-k5k/litellm-1-1-onboarding-chat)

- No => Then, No impact

### Are any other features going to be impacted? 

No. We treat this seriously, and don't plan on moving any other features to enterprise. Only new features will be added to it. 

Features like key creating, budget management, spend tracking, langfuse logging, admin ui, team-specific callbacks, etc. will all remain open source. 

### Can I still scale with just the open-source version? 

Yes! You will always be able to scale with just open-source litellm. 

### What about future features? 

We want to minimize movement from open-source (free) to enterprise (paid).

So, at the start if a feature seems suited for enterprise-only, we'll first put it there. If we're wrong, we'll move it into the open-source repo.   

### What does `enterprise/` folder mean? 

Features in this [public folder](https://github.com/BerriAI/litellm/tree/main/enterprise), will be under a commercial license  (to get one, just talk with us on a [call](https://calendly.com/d/4mp-gd3-k5k/litellm-1-1-onboarding-chat) / [email](mailto:info@berri.ai)  / [discord](https://discord.com/invite/wuPM9dRgDw)).

**Comment by nicornk** [comment]


**Reply by ghost** [reply]
Hey @nicornk that's totally fair feedback. 

I'd love to learn how you're using liteLLM, and see how we can help - https://calendly.com/d/4mp-gd3-k5k/litellm-1-1-onboarding-chat

If none of these times work, I can make anytime that works for you.

**Reply by ghost** [reply]
@nicornk dm'ed on LinkedIn as well. Would love to understand how you're using litellm.
